samples/config/start_config.json

```diff
@@ -16,7 +16,10 @@
         "bind_address": "127.0.0.1:8082",
         "published_address": "ccf.dummy.com:12346",
         "max_open_sessions_soft": 1000,
-        "max_open_sessions_hard": 1010
+        "max_open_sessions_hard": 1010,
+        "endorsement": {
+          "authority": "Node"
```

Comment:
@jumaffre do we also capitalise enum values elsewhere in the config? If not, I'd rather have the config look one way than arbitrarily enforce consistency with the C++ type on a one-off basis.

Reply:
Lower case: enclave.type, command.type, node_certificate.curve_id, logging.host_level, logging.format
Camel case: reconfiguration_type
All caps, but going away: consensus_type

Reply:
To avoid a mix of casing between enums exposed in OpenAPI and the config, esp. in cases where they are used in both, we decided to align with OpenAPI and rename all config enum values to be camel case. Therefore, this PR won't change since it uses that convention already.
@jumaffre